### PR TITLE
docs: sequence controller w/ clearer descriptions

### PR DESCRIPTION
**Pull request by stijnmoreels**
A proposal for a more dev/user-friendly description on the **Sequence Controller** by first going over the motivation and reason, and afterwards laying more the focus on the 'What' the component does instead of 'How' it does it.

### CODE REVIEW

**Comment by stijnmoreels** [comment]
@pim-simons , can you also give your opinion on this? Especially on the 'pseudo Logic App workflow' and the motivations. Hopefully it gets more clear for all of us. 😉
Thx in advance!

**Review by GoutsmitSam — COMMENTED** [review]
@stijnmoreels  The internal links to the different sequencecontroller actions don't work for me in the Deploy Preview page (for example https://deploy-preview-230--invictus-for-azure.netlify.app/#/framework/components/sequencecontroller?id=2-wait-for-sequence)

Apart from that, it seems nice and clear to me . Thanks!

**Comment by stijnmoreels** [comment]
> @stijnmoreels The internal links to the different sequencecontroller actions don't work for me in the Deploy Preview page (for example https://deploy-preview-230--invictus-for-azure.netlify.app/#/framework/components/sequencecontroller?id=2-wait-for-sequence)
> 
> Apart from that, it seems nice and clear to me . Thanks!

Fixed that by a heading underscore `_` as the headers start with a number (and Docusaurus didn't picked this up). Fixed this here: https://github.com/invictus-integration/docs-ifa/pull/230/commits/983960d4317dada7493fba060db4cde4fe6735ec

File: framework/components/sequencecontroller.md [hunk 1]
```diff
@@ -1,52 +1,115 @@
 # Sequence Controller
 
-## Overview
+## Motivation
 
-The Sequence controller will be mainly used to process LogicApp runs in order. The Sequence Controller exposes 3 actions:
+Some dependent external systems can't handle parallel message processing or requires a certain order of messages. Sending messages serially can be a solution, at the cost of a big performance impact. If the processing or sending of a single message is too big, serial is still not the solution.
```
Comment (pim-simons) [inline framework/components/sequencecontroller.md:5]:
```suggestion
Some dependent external systems can't handle parallel message processing or requires a certain order of messages. Sending messages serially (for example by setting the concurrency on a Logic App to `1`) can be a solution, at the cost of a big performance impact. If the processing or sending of a single message is too big, serial is still not the solution.
```

File: framework/components/sequencecontroller.md [hunk 2]
```diff
@@ -1,52 +1,115 @@
...
 
-|Name|Required|Sample Value|Description|
-| --- | :---: | --- | --- |
-|sequenceName|Yes|SampleSequence1|This is the name given to a sequence, this has to be unique per sequence when processing a chain of requests|
-|sequenceStart|Yes|1|This is the starting value of the sequence, this in reality is only required on the very first call to GetSequenceNumber but has no effect and should still be passed on subsequent calls|
+The **Sequence Controller** component is available as a HTTP endpoint in your Logic App workflow. To include sequence processing to your workflow, these three HTTP interaction tasks should normally be added:
+
+* [Get sequence number](#_1-get-sequence-number): allows the workflow to determine what current position it has in the sequence;
+* [Wait for sequence](#_2-wait-for-sequence): allows the workflow to wait its turn, doing the actual work in a **Control** task;
+* [Complete sequence](#3-complete-sequence): allows the workflow to signal that the next workflow in the sequence is up.
```
Comment (pim-simons) [inline framework/components/sequencecontroller.md:17]:
```suggestion
* [Complete sequence](#_3-complete-sequence): allows the workflow to signal that the next workflow in the sequence is up.
```

this link doesn't work yet 😉

File: framework/components/sequencecontroller.md [hunk 3]
```diff
@@ -1,52 +1,115 @@
...
+#### Logic App workflow example
+
+The following image shows a screenshot of how this HTTP call can be made in a Logic App workflow.
 
 > ![sequencecontroller-get-number-execution](../../images/seqcont-getseqnum.jpg)
 
-### WaitForExecution
+### 2. Wait for sequence
+
+The next step for the Logic App workflow to run in sequence, is to wait its turn. To facilitate this, the counter collected from the previous [Get sequence number](#1-get-sequence-number) step is required.
```
Comment (pim-simons) [inline framework/components/sequencecontroller.md:61]:
```suggestion
The next step for the Logic App workflow to run in sequence, is to wait its turn. To facilitate this, the counter collected from the previous [Get sequence number](#_1-get-sequence-number) step is required.
```

File: framework/components/sequencecontroller.md (hunk 1 above)
Comment (pim-simons) [inline framework/components/sequencecontroller.md:5]: I don't quite understand this line

> If the processing or sending of a single message is too big, serial is still not the solution.


I think a better example to give is the following real-life example from a customer.
A downstream application cannot handle multiple calls at the same time for the same orderid, by using the sequencecontroller we are able to make sure that a single call per orderid is done (by using the orderid in the sequencename) and we do not have the use the concurrency option for the Logic App which would dramatically slow down the processing for all messages.

**Review by pim-simons — CHANGES_REQUESTED** [review]
Some small comments, but looks good overall 🥇

File: framework/components/sequencecontroller.md (hunk 1 above)
Comment (stijnmoreels) [reply framework/components/sequencecontroller.md]: Thx a lot for this, Pim!
Added 'a story from the trenches' here: https://github.com/invictus-integration/docs-ifa/pull/230/commits/6b80f615e9f6d332d86998533ad6953f7839c06b

**Review by pim-simons — APPROVED** [review]

File: framework/components/sequencecontroller.md [hunk 4]
```diff
@@ -1,52 +1,119 @@
...
+
+![Pseudo Logic App workflow with Sequence Controller](../../images/framework/pseudo-logic-app-w-sequence-controller.png)
+
+### 1. Get sequence number
+
+First step for the Logic App workflow to run in sequence, is to take a number in the line. Doing this requires you to send a HTTP POST request to the `/api/GetSequenceNumber` endpoint of the deployed **Sequence Controller**.
+
+> 💡 This step can be circumvented in some advanced scenarios where you keep track of the order numbers and pass it yourself during the [Wait for sequence](#_2-wait-for-sequence).
+
+The most simple request body only contains the name of the sequence. This name can be seen as the transactional group that chain all related workflows. Assuming this is the first workflow, it will receive number 1 in the response body; the second workflow will receive number 2, and so forth.
```
Comment (stijnmoreels) [inline framework/components/sequencecontroller.md:35]: When no `sequenceStart` is defined, it currently returns `0` as the 'first' number, but there's a PR that adds `1` as the new default.

Comment (stijnmoreels) [reply framework/components/sequencecontroller.md:35]: Discussed separately: it is not required to go tht much in detail.

**Pull request merged**